src/test/a11y/test-suite.js

```diff
@@ -108,4 +100,32 @@ const runTests = async () => {
   }
 }
 
-runTests()
+const runILiveInScotlandTest = async (results) => {
+  try {
+    const { requestCookie, csrfToken } = await getSIDCookieAndCSRFToken(ENTER_DOB_URL)
+    const pa11y = pa11yWithSettings(IGNORE_RULES, { Cookie: requestCookie })
+    const formData = { '_csrf': csrfToken }
+
+    results.push(await pa11y(DO_YOU_LIVE_IN_SCOTLAND_URL))
+    await postFormData(DO_YOU_LIVE_IN_SCOTLAND_URL, { ...formData, doYouLiveInScotland: 'yes' }, requestCookie)
+
+    results.push(await pa11y(I_LIVE_IN_SCOTLAND_URL))
+  } catch (error) {
+    console.log(error)
+    process.exit(1)
+  }
+}
+
+const runAllTests = async () => {
+  try {
+    let results = []
+    await runEndToEndTest(results)
+    await runILiveInScotlandTest(results)
+    handleTestResults(results)
+  } catch (error) {
+    console.log(error)
```

Comment:
Should be console.error(), same for above.

Reply:
Ok, will do, but that's what you put there in your original commit Dave. :-)